lessons/13-server-rendering/README.md

```diff
@@ -50,7 +46,7 @@ module.exports = {
 
   module: {
     loaders: [
-      { test: /\.js$/, exclude: /node_modules/, loader: 'babel-loader?presets[]=es2015&presets[]=react' }
+      { test: /\.js$/, loader: 'babel-loader?presets[]=es2015&presets[]=react' }
```

Comment:
This exclusion is important so Babel won't run on all of your node_modules code, which can be very substantial.